numbagg/moving.py

```diff
@@ -1,8 +1,50 @@
 import numpy as np
+from numba import guvectorize, float64
 
 from .decorators import ndmoving
 
 
+@guvectorize(
+    [(float64[:], float64, float64[:])],
```

Comment:
One of the reasons this doesn't work with `ndmoving` is that the signature isn't `['float64,int64,float64']` as that currently requires. But we could make it abstract over that

Reply:
Yeah, it would be nice to make use of `ndmoving` so we can use the `axis` argument.

I think we would need to make the use/validation of the `window` argument optional.